schema/resolvers.test.js

```diff
@@ -1461,5 +1465,81 @@ describe("resolvers", () => {
         expect(res2.errors).toHaveLength(1);
       });
     });
+
+    describe("Numeric routing", () => {
+      it("should be able to create a routing rule for numeric answers", async () => {
+        const answer = await createNewAnswer(firstPage, "Currency");
+        await createNewRoutingRuleSet(firstPage.id);
+
+        const routingInfo = await getFullRoutingTree(firstPage);
+        const routingCondition = get(
+          routingInfo,
+          "data.questionPage.routingRuleSet.routingRules[0].conditions[0]"
+        );
+
+        expect(routingCondition).toMatchObject({
```

Comment:
These are default values? I can't see where they are defined in this PR.

Reply:
Equal is the default value, it would have been defined in the first implementation of routing, this one just adds other comparators without changing the default.